### PR TITLE
Refactor ACP function names to align with rest of API

### PR DESCRIPTION
**Pull request by NSeydoux**
- xxxForRule becomes xxx (e.g. hasPublicForRule -> hasPublic), and yyyForPolicy becomes yyy
- AccessControl becomes Control

# Checklist

- [X] All acceptance criteria are met.
- [X] Relevant documentation, if any, has been written/updated.
- [X] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: src/acp/policy.ts [hunk 1]
```diff
@@ -149,16 +149,13 @@ export function setPolicy(
...
  * ```
  *
- * Given a [[Policy]] and a set of [[AccessModes]], return a new Access Policy based on the given
- * Access Policy, but with the given Access Modes allowed on it.
+ * Given a [[Policy]] and a set of [[AccessModes]], return a new Policy based on the given
+ * Policy, but with the given Access Modes allowed on it.
  *
- * @param policy An Access Policy on which to set the modes to allow.
- * @param modes Modes to allow for this Access Policy.
+ * @param policy A Policy on which to set the modes to allow.
```
Comment (pmcb55) [inline src/acp/policy.ts:155]: 4 references to 'A Policy', all of which I'd change to 'The Policy'.

File: src/acp/policy.ts [hunk 2]
```diff
@@ -180,9 +177,9 @@ export function setAllowModesOnPolicy(
  *
  * Given a [[Policy]], return which [[AccessModes]] it allows.
  *
- * @param policy An Access Policy of which you want to know which Access Modes it allows.
+ * @param policy A Policy of which you want to know which Access Modes it allows.
```
Comment (pmcb55) [inline src/acp/policy.ts:180]:
```suggestion
 * @param policy The Policy for which you want to know the Access Modes it allows.
```

File: src/acp/policy.ts [hunk 3]
```diff
@@ -195,16 +192,13 @@ export function getAllowModesOnPolicy(policy: Policy): AccessModes {
...
  * ```
  *
- * Given a [[Policy]] and a set of [[AccessModes]], return a new Access Policy based on the given
- * Access Policy, but with the given Access Modes disallowed on it.
+ * Given a [[Policy]] and a set of [[AccessModes]], return a new Policy based on the given
+ * Policy, but with the given Access Modes disallowed on it.
  *
- * @param policy An Access Policy on which to set the modes to disallow.
- * @param modes Modes to disallow for this Access Policy.
+ * @param policy A Policy on which to set the modes to disallow.
```
Comment (pmcb55) [inline src/acp/policy.ts:198]:
```suggestion
 * @param policy The Policy on which to set the modes to disallow.
```

File: src/acp/policy.ts [hunk 4]
```diff
@@ -226,9 +220,9 @@ export function setDenyModesOnPolicy(
  *
  * Given a [[Policy]], return which [[AccessModes]] it disallows.
  *
- * @param policy An Access Policy of which you want to know which Access Modes it disallows.
+ * @param policy A Policy of which you want to know which Access Modes it disallows.
```
Comment (pmcb55) [inline src/acp/policy.ts:223]:
```suggestion
 * @param policy The Policy on which you want to know the Access Modes it disallows.
```

**Review by pmcb55 — APPROVED** [review]
LGTM - just really minor comment suggestions.

File: src/acp/policy.ts (hunk 1 above)
Comment (NSeydoux) [reply src/acp/policy.ts:155]:
```suggestion
 * @param policy The Policy on which to set the modes to allow.
```

File: src/acp/control.ts [hunk 5]
```diff
@@ -116,29 +116,29 @@ export type WithLinkedAcpAccessControl<
...
+  control = setIri(control, rdf.type, acp.AccessControl);
+  return control;
 }
 /**
  * ```{note} The Web Access Control specification is not yet finalised. As such, this
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Find an [[AccessControl]] with a given URL in a given Resource with an Access Control Resource.
+ * Find an \[\[Control\]\] with a given URL in a given Resource with an Access Control Resource.
```
Comment (Vinnl) [inline src/acp/control.ts:133]: Did something go wrong here with a search and replace? The same happens at a couple of other places below.

```suggestion
 * Find an [[Control]] with a given URL in a given Resource with an Access Control Resource.
```

**Review by Vinnl — APPROVED** [review]
I guess this is what we took years of education for :P

File: src/acp/control.ts (hunk 5 above)
Comment (NSeydoux) [reply src/acp/control.ts]: Exactly ^^. Fixed in c9d70605d4b499bd647259fce5594bef2c8a4718

**Review by pmcb55 — APPROVED** [review]
LGTM

**Pull request merged**